frontend_server/README.md

```diff
@@ -0,0 +1,35 @@
+Frontend Server
+==============
+
+Frontend server is simple wrapper around Dart Frontend. It is a Dart application
+that compiles Dart source into Dart Kernel binary(.dill-file).
+
+Frontend server runs in two modes:
+ - immediate mode, where Dart source file name is provided as command line
+ argument;
+ - interactive mode, where communication is happening over stdin/stdout.
+
+ Interactive mode instructions
+ ---
+- ```compile <input.dart>```
+ Compiles <input.dart> Dart source file with Dart Frontend, replies with ```result``` response.
+- ```recompile <boundary-key>```
+```<path/to/updated/file1.dart>```
+```<path/to/updated/file2.dart>```
+```...```
+```<boundary-key>```
+ Incrementally recompiles Dart program previously compiled in current session, taking into account changes in the listed files. Replies with ```result``` response.
+- ```accept```
+ Accepts results of incremental compilation, so that next recompilation request won't recompile previously reported changed files.
+- ```reject```
+ Rejects results of incremental compilation, so that next recompilation request will recompile previously reported changed files.
```

Comment:
Not sure I understand the comment here.
e.g: My application contains the following files (myapp.dart, mypart1.dart, mypart2.dart, mypart3.dart)
Step 1 (compile myapp.dart)
Step 2 (recompile mypart2.dart, mypart3.dart)
Step 3 (reject)
Step 4 (recompile mypart2.dart mypart1.dart) 
In Step 4 it should be recompiling only mypart2.dart and mypart1.dart correct. mypart3.dart which was previously reported as changed is not reported now because i discarded all the changes I made.